cli/src/main.rs

```diff
@@ -9,20 +10,25 @@ fn main() {
         .version(crate_version!())
         .author(crate_authors!())
         .setting(AppSettings::ArgRequiredElseHelp)
-        .subcommand(login_sub_command());
+        .subcommand(login_sub_command())
+        .subcommand(package_sub_command());
 
     let matches = app.get_matches();
     match matches.subcommand_name() {
         Some("login") => println!("loginnnn",), // login
+        Some("package") => package::create_package("./".to_string(), "/tmp/package.tar.gz".to_string()), // login
```

Comment:
i would assume we want to directly stream it, might be premature optimization, but i dont think a file is really needed.

Reply:
> i would assume we want to directly stream it, might be premature optimization, but i dont think a file is really needed.

Definitely agree, as I mentioned I saved it to a file as it helped me during the development of this, as a file is easier to debug and verify the behavior is what we want. (From outside the program)
I think we should keep it for debugging purposes, and once we know whats the interaction between the artifact and publishing, we should refactor this appropriately